core/src/main/java/discord4j/core/spec/MessageCreateSpec.java

```diff
@@ -57,17 +56,153 @@
-        this.content = content;
+        this.contentBuilder.setLength(0);
+        this.contentBuilder.append(content);
+        return this;
+    }
+
+    /**
+     * Gets the specs contents.
+     *
+     * @return Current contents of this spec.
+     */
+    public String getContent() {
+        return contentBuilder.toString();
+    }
+
+
+    /**
+     * Resets the specs contents.
+     *
+     * @return This spec.
+     */
+    public MessageCreateSpec resetContent() {
+        this.contentBuilder.setLength(0);
+        return this;
+    }
+
+    /**
+     * Resets the embed.
+     *
+     * @return This spec.
+     */
+    public MessageCreateSpec resetEmbed() {
+        embed = null;
+        return this;
+    }
+
+    /**
+     * Resets the files.
+     *
+     * @return This spec.
+     */
+    public MessageCreateSpec resetFiles() {
+        files = null;
+        return this;
+    }
+
+    /**
+     * Resets allowed mentions.
+     *
+     * @return This spec.
+     */
+    public MessageCreateSpec resetAllowedMentions() {
+        allowedMentionsData = null;
+        return this;
+    }
+
+    /**
+     * Resets this spec.
+     *
+     * @return This spec.
+     */
+    public MessageCreateSpec reset() {
+        return this
+            .resetAllowedMentions()
+            .resetContent()
+            .resetEmbed()
+            .resetFiles()
+            .resetMessageReference();
+    }
+
+    /**
+     * Resets the message reference.
+     *
+     * @return This spec.
+     */
+    public MessageCreateSpec resetMessageReference() {
+        messageReferenceData = null;
+        return this;
+    }
+
+    /**
+     * Adds formatted text to this spec.
+     * @param content The content to be formatted and added.
+     * @param formats {@link #MARKDOWN_ITALIC}, {@link #MARKDOWN_BOLD}, {@link #MARKDOWN_UNDERLINE} or {@link #MARKDOWN_CODELINE}
+     * @return This spec.
+     */
+    public MessageCreateSpec appendFormatted(String content, byte... formats) {
+        ArrayList<String> tags = new ArrayList<>();
+        for (byte format : formats) {
+            String tag = getMarkdownTag(format);
+            if (tag == null) {
+                throw new IllegalArgumentException("Invalid markdown format!");
+            }
+            tags.add(tag);
+        }
+        tags.forEach(contentBuilder::append);
+        contentBuilder.append(content);
+        tags.forEach(contentBuilder::append);
+        return this;
+    }
+
+
+    /**
+     * Adds a code block to this spec.
+     *
+     * @param content  The content of the code block.
+     * @param language The language of the code block. (If null, no languages.)
+     * @return This spec.
+     */
+    public MessageCreateSpec appendCodeBlock(String content, @Nullable String language) {
+        contentBuilder
+            .append("```");
+        if (language != null) {
+            contentBuilder.append("\n").append(language);
```

Comment:
Shouldn't `append(language)` be before `append('\n')` ->  "```<language>\n"

Reply:
Oh. True.

Reply:
Fixed it with the new commit.